### ISSUE TITLE
Read endpoints from configuration for current deployment/environment/venue

### ISSUE THREAD

**Issue by rtapella** [unity-sds/unity-sds-portal#3]
- read config as in https://github.com/unity-sds/unity-cs-infra/issues/17
- this helps build the links in the navbar in https://github.com/unity-sds/unity-project-management/issues/74

**Comment by rtapella** [comment]
We should probably set up a hard-coded configuration file here that Has the same info we’ll need/get from envvars/settings-database. E.g. a way to check whether the logged-in person has access to something (so we can only show what they have access to, or otherwise indicate that they do not have access)

**Comment by rtapella** [comment]
put Project and Venue in the navbar but they're static (it only knows about "this" venue)

**Comment by rtapella** [comment]
part of https://github.com/unity-sds/unity-project-management/issues/74

**Comment by rtapella** [comment]
meeting to discuss on Feb 27

**Comment by GodwinShen** [comment]
@rtapella is there any update on this ticket?  Also, it looks like this is a blocker for https://github.com/unity-sds/unity-sds-portal/issues/3

**Comment by rtapella** [comment]
potentially for an initial pass, this could use the health-status to see which apps are available and even indicate any that are "down" in the navigation menu

**Comment by rtapella** [comment]
this issue will be done when reading venue and other config from the environment so that Terraform can push the right config to the navbar.

navigation-bar customization based on available apps is [#20](https://github.com/unity-sds/unity-ui/issues/20)

**Comment by rtapella** [comment]
@anilnatha how does the new work dovetail with this? should this issue be closed and a new one created to track the config reading from a venue?

**Comment by rtapella** [comment]
@anilnatha if this should be open, please move to the correct sprint

**Comment by GodwinShen** [comment]
@anilnatha and @rtapella ping for status.  looks like this was supposed to be done in 24.2.1 but maybe y'all didn't settle on the exact sprint to do it in.

**Comment by anilnatha** [comment]
@GodwinShen Yup, working on this in this sprint. Tagged up with Rob about the marketplace tasks for this sprint yesterday and this is related to that.

**Comment by GodwinShen** [comment]
@anilnatha and @rtapella can we get a quick update on this ticket?
